SpywareFilter/sections/tracking_servers_firstparty.txt

```diff
@@ -1420,6 +1420,8 @@
 ||hodor-collect.arabam.com^
 ||horizon-track.globo.com^
 ||horizon.globo.com^
+@@||googletagmanager.com^$domain=ge.globo.com
+@@||globo.com^$cookie=_ga
```

Comment:
Wrong file, add them to `allowlist.txt` with the task issue.
Besides, other rules are needed on my end.
```adblock
! https://github.com/AdguardTeam/AdguardFilters/issues/181699
@@||globo.com^$cookie=_ga
@@||globo.com^$cookie=/^_ga_/
@@||googletagmanager.com/gtm.js$domain=globo.com
@@||googletagmanager.com/gtag/$domain=globo.com
```

Reply:
As an alternative (or additional) solution we could use scriptlets instead of unblocking `googletagmanager.com`:
```adblock
globo.com#%#//scriptlet('trusted-set-cookie', '_ga', '1.1')
globo.com#%#//scriptlet('trusted-set-constant', 'libAnalytics', '{"status":{"dataAvailable":false}}')
globo.com#%#//scriptlet('trusted-set-constant', 'dataLayer', '[{"bd_suser_provider": "1", "bd_suser_code": "1"}]')
```
Seems to works fine on my end.

Reply:
@AdamWr yep, works fine on [the link](https://ge.globo.com/futebol/times/palmeiras/noticia/2024/06/18/dudu-volta-aos-treinos-no-palmeiras-e-fica-a-disposicao-de-abel-ferreira.ghtml), but not [here](https://g1.globo.com/pe/pernambuco/noticia/2024/06/19/stalker-de-debora-falabella-teve-primeiro-surto-esquizofrenico-em-2004-passou-21-dias-internada-em-paris-diz-mae.ghtml) on my end. If we talk about other platforms, the domain should be added in this rule.
```adblock
||google-analytics.com/analytics.js$script,redirect=google-analytics,domain=~olx.*|~banki.ru|~bigc.co.th|~sonos.com
```
Could you also check this?

Reply:
I checked only the scriptlets, but they do work with the cookie exceptions.

Reply:
I guess we can add the rules.
```adblock
! https://github.com/AdguardTeam/AdguardFilters/issues/181699
@@||globo.com^$cookie=_ga
@@||globo.com^$cookie=/^_ga_/
globo.com#%#//scriptlet('trusted-set-cookie', '_ga', '1.1')
globo.com#%#//scriptlet('trusted-set-constant', 'libAnalytics', '{"status":{"dataAvailable":false}}')
globo.com#%#//scriptlet('trusted-set-constant', 'dataLayer', '[{"bd_suser_provider": "1", "bd_suser_code": "1"}]')
!#if (adguard_ext_safari || adguard_app_ios || adguard_ext_android_cb || ext_ublock)
@@||googletagmanager.com/gtm.js$domain=globo.com
@@||googletagmanager.com/gtag/$domain=globo.com
!#endif